include/ceed-backend.h

```diff
@@ -26,6 +26,7 @@
 
 #define CEED_MAX_RESOURCE_LEN 1024
 #define CEED_ALIGN 64
+#define CEED_COMPOSITE_MAX 16
```

Comment:
I think we may want to do something like this for the total number of input/output fields and other constants.